### PR TITLE
Fix Room and equipment mailboxes link

### PR DESCRIPTION
**Pull request by niclaszll**
Adding a missing bracket.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/recipients/room-mailboxes.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ If you want to learn about the types of recipients that are available in Exchang
 
 - For information about keyboard shortcuts that may apply to the procedures in this topic, see [Keyboard shortcuts in the Exchange admin center](../about-documentation/exchange-admin-center-keyboard-shortcuts.md).
 
-- If you're using room or equipment mailboxes in Microsoft 365 or Office 365, see [Room and equipment mailboxes(https://docs.microsoft.com/microsoft-365/admin/manage/room-and-equipment-mailboxes) for more information.
+- If you're using room or equipment mailboxes in Microsoft 365 or Office 365, see [Room and equipment mailboxes](https://docs.microsoft.com/microsoft-365/admin/manage/room-and-equipment-mailboxes) for more information.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/recipients/room-mailboxes.md:47]:
```suggestion
- If you're using room or equipment mailboxes in Microsoft 365 or Office 365, see [Room and equipment mailboxes](/microsoft-365/admin/manage/room-and-equipment-mailboxes) for more information.
```

File: Exchange/ExchangeServer/recipients/room-mailboxes.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ If you want to learn about the types of recipients that are available in Exchang
...
 
 > [!IMPORTANT]
 > If you're running Exchange 2013 in a hybrid scenario, make sure you create the room mailboxes in the appropriate place. Create your room mailboxes for your on-premises organization on-premises, and room mailboxes for Exchange Online should be created in the cloud. > Having problems? Ask for help in the Exchange forums. Visit the forums at: [Exchange Server](https://social.technet.microsoft.com/forums/office/home?category=exchangeserver), [Exchange Online](/answers/topics/office-exchange-server-itpro.html), or [Exchange Online Protection](https://social.technet.microsoft.com/forums/forefront/home?forum=FOPE).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/recipients/room-mailboxes.md:50]:
```suggestion
> If you're running Exchange 2013 in a hybrid scenario, make sure you create the room mailboxes in the appropriate place. Create your room mailboxes for your on-premises organization on-premises, and room mailboxes for Exchange Online should be created in the cloud.
> Having problems? Ask for help in the Exchange forums. Visit the forums at: [Exchange Server](https://social.technet.microsoft.com/forums/office/home?category=exchangeserver), [Exchange Online](/answers/topics/office-exchange-server-itpro.html), or [Exchange Online Protection](https://social.technet.microsoft.com/forums/forefront/home?forum=FOPE).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @niclaszll, could you please also commit my additional suggested changes? Many thanks.

**Comment by niclaszll** [comment]
Added your suggested changes @JohanFreelancer9, thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**